src/main/java/org/folio/linked/data/integration/kafka/KafkaSenderFolio.java

```diff
@@ -135,7 +136,7 @@ private void sendDelete(Long id) {
   }
 
   private String getTenantTopicName(String tenantId) {
-    return String.format("%s.%s.%s", getFolioEnvName(), tenantId, initialBibframeIndexTopicName);
+    return KafkaUtils.getTenantTopicName(initialBibframeIndexTopicName, getFolioEnvName(), tenantId);
```

Comment:
`KafkaUtils.getTenantTopicName` has the logic to return the correct topic based on the FOLIO cluster setup.

Reply:
we also could omit 'getFolioEnvName()' argument, because it's provided by default in org.folio.spring.tools.kafka.KafkaUtils#getTenantTopicName(java.lang.String, java.lang.String)

Reply:
Done